github_changelog_generator.gemspec

```diff
@@ -8,7 +8,7 @@ Gem::Specification.new do |spec|
   spec.name               = "github_changelog_generator"
   spec.version            = GitHubChangelogGenerator::VERSION
 
-  spec.required_ruby_version = ">= 2.5.0"
+  spec.required_ruby_version = ">= 3.2.0"
```

Comment:
Oh, can we keep this to 3.0.0? This is the required lowest version that's allowed to run the software.

Reply:
Sure. I'll make some adjustments since it'll affect the cops enabled.